gui/wxpython/gis_set.py

```diff
@@ -530,6 +535,7 @@ def SuggestDatabase(self):
                 'your home directory. '
                 'Press Browse button to select the directory.'))
 
+
```

Comment:
Please remove this extra change. Also in PEP8, there is only one empty line between methods.